source/_includes/custom/after_footer.html

```diff
@@ -1,3 +1,53 @@
 {% comment %}
-  Add content to be output at the bottom of each page. (You might use this for analytics scripts, for example)
 {% endcomment %}
+
+<script type="text/javascript">
+  // made header sticky
+  var sticky = new Waypoint.Sticky({
+    element: $("#banner"),
+    offset: -100
+  })
+
+  {% if page.opensource == true %}
```

Comment:
Hmm I see how that could be tricky. I'm honestly not so concerned about it b/c this is only slightly less-nice to maintain than a separate js file. The real downside is the fact that we include a bunch of separate script tags + this embedded JS. What we ideally want is a build step that bundles up all of the JS dependencies (backbone,etc) and minifies it. I think it would be worth addressing this embedded code along with that task. So for now this seems alright to me.